vnext/ReactUWP/Modules/Animated/AdditionAnimatedNode.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "AdditionAnimatedNode.h"
+
+namespace react {
+  namespace uwp {
+    AdditionAnimatedNode::AdditionAnimatedNode(int64_t tag, const folly::dynamic& config, const std::shared_ptr<NativeAnimatedNodeManager>& manager) : ValueAnimatedNode(tag, config, manager)
+    {
+      for (auto inputNode : config.find(s_inputName).dereference().second)
+      {
+        m_inputNodes.insert(inputNode.asInt());
```

Comment:
we've tried to remove asInt() from the repo because it throws exceptions when the value is not a whole number, static_cast<int64_t>(inputNode.asDouble())